.gitignore

```diff
@@ -62,3 +62,4 @@ resources/examples/airflow/logs/*
 !airbyte-webapp/src/packages/cloud/data
 /bin/
 /**/bin/
```

Comment:
Why are `/bin/` and `/**/bin/` ignored in the first place?

Reply:
> Why are `/bin/` and `/**/bin/` ignored in the first place?

Good question, I have no idea. I choose the option with the lower risk based on what I saw here, but ideally we should remove the exclusion.

Reply:
If you build https://github.com/airbytehq/airbyte/pull/7005 (which added this change) does it create binaries that should be ignored? If so, can you move the ignore to that level instead of global.

Reply:
The build passed without any file changes so it should be ok to remove the ignore. I didn't manage to run the buildlocally though

Reply:
Did you build this connector specifically?

Reply:
yes, it is not generating anything in any bin folder.